twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -164,11 +181,12 @@ def _compile_to_scala(ctx, label, compile_thrifts, include_thrifts, jar_output):
             # always add finagle option which is a no-op if there are no services
             # we could put "include_services" on thrift_info, if needed
             "--with-finagle",
+            "--language={}".format(language),
         ]),
     )
 
     argfile = ctx.actions.declare_file(
-        "%s_worker_input" % label.name,
+        "{}_{}_worker_input".format(label.name, language),
```

Comment:
this is needed if we have two targets with the same label but for java/scala scrooge? I think this isn't legal bazel wise to have targets with the same name for different rules.
If this serves a different purpose I'd love to know.
I got to it btw from trying to find a test that needs this and couldn't find it so if it does exist I'd love to know

Reply:
In this code, `target` will be a thrift target, and we may need to generate both java and scala out of it. The same reason to parametrize by language here: https://github.com/bazelbuild/rules_scala/pull/1055/files/c346443ab84f4f60b6178bdd2899102951d3519b#diff-4fe70c7ad76a7189b88d4792f15a77ffR336

Reply:
do we have an example of trying to generate both scala and java from a single thrift target?
I think we do but I also think that I changed this line back and nothing broke. 
If you can validate it I'd be grateful.